content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -147,7 +147,7 @@ If your cluster's control plane runs the kube-apiserver as a Pod, remember to mo
 to the location of the policy file and log file, so that audit records are persisted. For example:
```

Comment:
Maybe also this?
```suggestion
for the policy file for the directory that contains the log file, so that audit records are persisted.
For the log output you mount a directory, and not just a log file, to allow for log rotation.

For example:
```